js/index.js

```diff
@@ -39,4 +39,46 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+logo.src = 'img/logo.png';
+
+const nav = document.querySelectorAll('nav');
+const aTag = document.querySelectorAll('a');
+const ctaSec = document.querySelectorAll('.cta');
+const mainCon = document.querySelectorAll('.main-content');
+const ctaImg = document.getElementById("cta-img");
+const midImg = document.getElementById('middle-img');
+const contact = document.querySelectorAll('.contact');
+const foot = document.querySelectorAll('footer');
+const btn = document.querySelectorAll('button');
+const h1 = document.querySelectorAll('h1');
+const h4 = document.querySelectorAll('h4');
+const p = document.querySelectorAll('p');
+
+
```

Comment:
You have the right idea for selecting your links, but here's how you can fix this: